flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/table/KafkaDynamicTableFactoryTest.java

```diff
@@ -851,6 +851,17 @@ public void testPrimaryKeyValidation() {
             assertEquals(error, t.getCause().getMessage());
         }
 
+        try {
+            createTableSink(pkSchema, getKeyValueOptions());
+            fail();
+        } catch (Throwable t) {
```

Comment:
Can you quickly explain why is it enough to only add this assertion to an existing test over adding a new test?

Reply:
The test was already testing the primary key validation in this scenario. The only difference is that it previously only tested the validation for when 'format' is set in the options (`getBasicOptions`). I added this to also test the validation in case 'value.format' is set (`getKeyValueOptions`)